app/components/file/details/file-details.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from "@angular/core";
+
+import { commonModules } from "app/common";
+import { NodeBrowseModule } from "app/components/node/browse";
+import { PoolGraphsModule } from "app/components/pool/graphs";
```

Comment:
Why do we need the pools Graphs module here? Same goes for the start task module.